man/systemd-coredump.xml

```diff
@@ -88,6 +88,42 @@
       metadata fields in the same way it does for core dumps received from the kernel. In this mode, no core
       dump is stored in the journal.</para>
     </refsect2>
+
+    <refsect2>
+      <title>Core dumps in Containers/Namespaces</title>
+
+      <para>The <filename>systemd-coredump@.service</filename> service will automatically attempt to extract
+      a stacktrace from any core dump as it is generated. For this stacktrace symbols will be resolved based
```

Comment:
I don't like this formulation, because we'll generate a stacktrace even if the core dump is not saved. Maybe say "will attempt to extract a stacktrace from a processes".